src/zero/api/handlers.clj

```diff
@@ -43,7 +43,7 @@
   "Add JWT to REQUEST when it contains a broadinstitute.org account.
   Otherwise redirect HANDLER to the LANDING-URI."
   [handler]
-  (let [{:keys [client-id landing-uri]} (:google oauth2-profiles)]
+  (let [{:keys [client-id]} (:google oauth2-profiles)]
```

Comment:
Does your prior PR break Google OAuth?

Reply:
No, Google OAuth works through the UI like it has been. But when I send an API request outside of the UI, where would I get that token from?

Reply:
So that's why I created a JWT from a service account and sent it with the authorization headers...but this authorize function wasn't checking that 🤔

Reply:
Thank you for checking.
Probably should remove `LANDING-URI` from the docstring then.
And maybe replace the top `let` with a `get-in`?

Reply:
Ahh I forgot to update the docstring 🤦‍♀

Reply:
I forget all the time. Only noticed on the 3rd read, FWIW.